source/tests/system/nirfmxnr_driver_api_tests.cpp

```diff
@@ -21,13 +21,13 @@ namespace tests {
 namespace system {
 namespace {
 
-const auto PXI_5663E = "5663E";
-const auto NR_SYNC_FAILURE = 684300;
-const auto NR_SYNC_FAILURE_STR = "Failed to synchronize to the signal";
-const auto IVI_ERROR_INVALID_VALUE = -1074135024;
-const auto IVI_ERROR_INVALID_VALUE_STR = "Invalid value for parameter or property";
-const auto INCORRECT_TYPE_ERROR_CODE = -380251;
-const auto INCORRECT_TYPE_ERROR_STR = "Incorrect data type specified";
+constexpr auto PXI_5663E = "5663E";
+constexpr auto NR_SYNC_FAILURE_WARNING = 684300;
+constexpr auto NR_SYNC_FAILURE_WARNING_STR = "Failed to synchronize to the signal";
+constexpr auto IVI_ERROR_INVALID_VALUE = -1074135024;
```

Comment:
These two should be re-arranged to match the new naming scheme: `IVI_INVALID_VALUE_ERROR` and `IVI_INVALID_VALUE_ERROR_STR`

Reply:
I left this one alone because the [normal error constant](http://ezerr/search/eng?q=-1074135024) that it's based on already has ERROR in the name. But if you think I should change it anyway, I can do that.

Reply:
Yeah, I'd go ahead and change it just so we're internally consistent. We can look up by the error code if needed so I'm not worried about discrepancy in names between here and the normal error constants.